### PR TITLE
Feat: Video listing

### PR DESCRIPTION
**Pull request by manojncsoni007**
Description:
- Added Video Listing page
- Added filtered video utilities
- Added CategoryChip Components

Screenshot
image

### CODE REVIEW

File: src/components/Categorychip/CategoryChip.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.category-chip{
+    border: 1px solid #a19f9f;
+    border-radius: 5rem;
+    padding: 0.5rem 1rem;
+    cursor: pointer;
+    margin: 0 0.5rem;
+}
+
+.category-chip:hover{
+    background-color: lightgray;
```
Comment (arpit01923) [inline src/components/Categorychip/CategoryChip.css:10]: use variable names'

File: src/components/Categorychip/CategoryChip.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useVideo } from '../../context/video-context';
```
Comment (arpit01923) [inline src/components/Categorychip/CategoryChip.jsx:2]: try absolute paths

File: src/pages/VideoListing/VideoListing.jsx [hunk 3]
```diff
@@ -1,9 +1,30 @@
 import React from 'react'
+import './VideoListing.css'
+import { CategoryChip, Sidebar, VideoCard } from '../../components'
```
Comment (arpit01923) [inline src/pages/VideoListing/VideoListing.jsx:3]: try absolute paths

**Review by arpit01923 — APPROVED** [review]
- clean UI
- good folder structure
- one suggestion, try absolute paths

**Review by shashikantmaurya0007 — APPROVED** [review]
clean UI
good folder structure
one suggestion, try absolute paths

File: src/components/Categorychip/CategoryChip.css (hunk 1 above)
Comment (manojncsoni007) [reply src/components/Categorychip/CategoryChip.css:10]: will do

**Pull request merged**